src/main/webapp/app/entities/patient/patient-details.vue

```diff
@@ -1,89 +1,175 @@
         </button>
-        <router-link v-if="patient.id" :to="{ name: 'PatientEdit', params: { patientId: patient.id } }" custom v-slot="{ navigate }">
-          <button @click="navigate" class="btn btn-primary">
-            <font-awesome-icon icon="pencil-alt"></font-awesome-icon>&nbsp;<span v-text="t$('entity.action.edit')"></span>
-          </button>
-        </router-link>
+      </router-link>
+    </div>
+    <div class="col-6 py-1">
+      <font-awesome-icon :icon="['fas', 'cake-candles']" />
+      <span class="h6">{{ patient.dateDeNaissance }}</span>
+    </div>
+    <div class="col-6 py-1">
+      <font-awesome-icon :icon="['fas', 'genderless']" />
+      <span class="h6">{{ patient.sexe }}</span>
+    </div>
+    <div class="col-6 py-1">
+      <font-awesome-icon :icon="['fas', 'location-dot']" />
+      <span class="h6">Chambre {{ patient.numChambre }}</span>
+    </div>
+    <div class="col-6 py-1">
+      <font-awesome-icon :icon="['fas', 'arrows-up-down']" />
+      <span class="h6">{{ patient.taille }}cm</span>
+    </div>
+    <div class="col-6 py-1">
+      <font-awesome-icon :icon="['fas', 'door-open']" />
+      <span class="h6">{{ patient.dateArrivee }}</span>
+    </div>
+  </div>
+
+  <div class="row justify-content-center mt-5">
+    <div class="col-12">
+      <div class="card">
+        <div class="card-body">
+          <h4 class="card-title" v-text="t$('g2EcomApp.patient.infoComplementaires')"></h4>
+          <p class="card-text">
+            {{ patient.infoComplementaires }}
+          </p>
+        </div>
       </div>
     </div>
   </div>
+  <div class="row justify-content-center text-center mt-5">
+    <div class="col-4" v-if="poidsPatient.length > 0">
+      <div class="card">
+        <h6 class="card-header" v-text="t$('g2EcomApp.patient.poids')"></h6>
+        <div class="card-body">
+          <h5>{{ poidsPatient[poidsPatient.length - 1].valeur }}</h5>
+        </div>
+      </div>
+    </div>
+    <div class="col-4" v-if="EPAPatient.length > 0">
+      <div class="card">
+        <h6 class="card-header" v-text="t$('g2EcomApp.patient.EPA')"></h6>
+        <div class="card-body">
+          <h5>{{ EPAPatient[EPAPatient.length - 1].valeur }}</h5>
+        </div>
+      </div>
+    </div>
+    <div class="col-4">
+      <div class="card">
+        <h6 class="card-header" v-text="t$('g2EcomApp.patient.IMC')"></h6>
+        <div class="card-body">
+          <h5>{{ patientIMC }}</h5>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row justify-content-center text-center mt-5">
+    <div class="col" v-if="weightChartLoaded">
+      <Line id="my-chart-id" :options="chartOptions" :data="weightChartData" />
+    </div>
+    <div class="col" v-if="EPAChartLoaded">
+      <Line id="my-chart-id" :options="chartOptions" :data="EPAChartData" />
+    </div>
+  </div>
+  <!--    <div class="col-8">-->
```

Comment:
commented code should be removed, git is there so that we can access this code later if we want to come back to it